src/mono/wasm/runtime/library_mono.js

```diff
@@ -1904,6 +1904,9 @@ var MonoSupportLib = {
 
 			if (invariantMode)
 				this.mono_wasm_setenv ("DOTNET_SYSTEM_GLOBALIZATION_INVARIANT", "1");
+			
+			// Set globalization mode to PredefinedOnly
+			this.mono_wasm_setenv ("DOTNET_SYSTEM_GLOBALIZATION_PREDEFINED_ONLY", "1");
```

Comment:
Why is it env variable if it's always set?

Reply:
Just leaving the option open if we don't want it always set.

Reply:
This won't be easy to change and it'll cost us on size. I think we should just compile the setting in

Reply:
We want it always set for wasm but I'm not sure if other OS want this option by default. @tarekgh

Reply:
outside WASM, we *shouldn't* turn this option by default. Whoever need it would need to opt-in to get it.

Reply:
We could do this by setting a default value for the flag and making that default value be true for wasm based on an OS check when you are setting the `PredefinedCulturesOnly` flag.

Reply:
Here is a place where we do something similar for WASM: https://github.com/dotnet/runtime/blob/e6105a1c752e065f1d7683d1e0903c81dee29e56/src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Unix.cs#L26

So we could do something like: 
```cs
GetSwitchValue("System.Globalization.PredefinedCulturesOnly", "DOTNET_SYSTEM_GLOBALIZATION_PREDEFINED_CULTURES_ONLY") || OperatingSystem.IsBrowser()
```